tests/conftest.py

```diff
@@ -45,4 +45,16 @@ def mock_openstack_base():
         "openstack_mcp_server.tools.base.get_openstack_conn",
         return_value=mock_conn
     ) as mock_func:
-        yield mock_conn
\ No newline at end of file
+        yield mock_conn
+
+@pytest.fixture
+def mock_openstack_connect_neutron():
+    """Mock openstack.connect function for neutron_tools."""
+    mock_conn = Mock()
+
+    with patch(
+        "openstack_mcp_server.tools.neutron_tools.openstack.connect",
+        return_value=mock_conn
+    ) as mock_func:
```

Comment:
죄송합니다 제가  as mock_func를 써서 쓰셨겠군요 

ruff rule에서 잡힙니다... 
```suggestion
    ):
```